dnf/yum/rpmtrans.py

```diff
@@ -129,15 +130,19 @@ def __init__(self):
                        self.PKG_OBSOLETE: _('Obsoleting'),
                        self.PKG_REINSTALL: _('Reinstalling'),
                        self.PKG_UPGRADE: _('Upgrading'),
-                       self.PKG_VERIFY: _('Verifying')}
+                       self.PKG_VERIFY: _('Verifying'),
+                       self.TRANS_PREPARATION: _('Preparing'),
```

Comment:
Originally I tried  "Preparing transaction" but during testing I reverted it, because output didn't look nice. I would prefer to keep short expression of action.